tests/conftest.py

```diff
@@ -62,7 +63,39 @@ def hydrocron_dynamo_instance(request, dynamo_test_proc):
         ProvisionedThroughput={
             'ReadCapacityUnits': 10,
             'WriteCapacityUnits': 10
-        }
+        },
+        GlobalSecondaryIndexes=[
```

Comment:
Does adding a GSR to an existing table have any implications we need to be aware of for deployment?

Reply:
This is adding a GSI to the test database which is already implemented in the various cloud venues. It does not impact the current unit tests and would not be something that gets deployed to any of the venues.